v0/destinations/fb/transform.js

```diff
@@ -220,16 +220,11 @@ function processEventTypeGeneric(message, baseEvent, fbEventName) {
 }
 
 function responseBuilderSimple(message, payload, destination) {
-  const requestConfig = {
-    requestFormat: "FORM",
-    requestMethod: "POST"
-  };
-
   const { appID } = destination.Config;
 
   // "https://graph.facebook.com/v3.3/644758479345539/activities?access_token=644758479345539|748924e2713a7f04e0e72c37e336c2bd"
 
-  const endpoint = `https://graph.facebook.com/v3.3/${appID}/activities`;
+  const endpoint = `https://graph.facebook.com/v13.0/${appID}/activities`;
```

Comment:
Have you validated sending the event to the destination as it is a long jump in respect of versions?

Reply:
+1

Reply:
Yes